detect-pr-changes.sh

```diff
@@ -67,16 +67,19 @@ done
 
 echo ""
 
+releasedApis=$apis
```

Comment:
A possibly-simpler option would be to create a new empty array, and add to it in the "else" block. But this is fine.